_episodes/03-create.md

```diff
@@ -14,12 +14,12 @@ Once Git is configured, we can start using it.  Let's create a
 directory for our work and then move into that directory:
 
 ~~~
-$ mkdir planets
-$ cd planets
+$ mkdir metadata
```

Comment:
We may want to highlight that people need to **not** be inside the `cats-as-metadata-master` folder they download/unzip in the setup. Maybe a note above to `cd Desktop` above or otherwise confirm they are in Desktop (pwd)?

Reply:
Actually, we may be able to 💥 this section since we're already having them `init` in the `cats-as-data`